src/Microsoft.ML.StandardLearners/Standard/LinearPredictor.cs

```diff
@@ -44,6 +44,7 @@ public abstract class LinearPredictor : PredictorBase<Float>,
         ICanSaveInTextFormat,
         ICanSaveInSourceCode,
         ICanSaveModel,
+        ICanGetSummaryAsIDataView,
```

Comment:
>ICanGetSummaryAsIDataView [](start = 8, length = 25)

Thanks Gani for making this change. Please take a look at these two classes inheriting from this class: LinearBinaryPredictor and LinearRegressionPredictor. They implement an interface called ICanGetSummaryAsIRow which this class should implement instead of ICanGetSummaryAsDataView.

Reply:
thx, this is done